src/core/jpg.js

```diff
@@ -332,6 +332,7 @@ var JpegImage = (function JpegImageClosure() {
     if (!resetInterval) {
       resetInterval = mcuExpected;
     }
+    var nonDefaultResetInterval = resetInterval !== mcuExpected;
```

Comment:
Can this logic here be expressed by replacing new code and the if above to `resetInterval = resetInterval ? Math.min(mcuExpected, resetInterval) : mcuExpected;`?